projects/packages/videopress/src/client/block-editor/blocks/video/components/color-panel/index.native.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import { useNavigation } from '@react-navigation/native';
+import { PanelColorSettings } from '@wordpress/block-editor';
+import { PanelBody, ToggleControl, BottomSheet } from '@wordpress/components';
+import { useDebounce } from '@wordpress/compose';
+import { useCallback, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronRight } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import { VideoBlockColorAttributesProps, VideoControlProps } from '../../types';
+/**
+ * Types
+ */
+import type React from 'react';
+
+/**
+ * React component that renders the playback bar color settings panel.
+ *
+ * @param {VideoControlProps} props - Component properties.
+ * @returns {React.ReactElement}    - Playback bar color settings panel.
+ */
+export default function ColorPanel( { attributes, setAttributes }: VideoControlProps ) {
+	const [ showSubSheet, setShowSubSheet ] = useState( false );
+	const navigation = useNavigation();
+
+	const goBack = useCallback( () => {
+		setShowSubSheet( false );
+		navigation.goBack();
+	}, [] );
+
+	const openSubSheet = useCallback( () => {
+		navigation.navigate( BottomSheet.SubSheet.screenName );
+		setShowSubSheet( true );
+	}, [] );
+
+	const { useAverageColor, seekbarColor, seekbarLoadingColor, seekbarPlayedColor } = attributes;
+
+	const initialColorState: VideoBlockColorAttributesProps = {
+		seekbarPlayedColor,
+		seekbarLoadingColor,
+		seekbarColor,
+	};
+
+	const [ colors, setColorsState ] = useState( initialColorState );
+
+	const debouncedSetColors = useDebounce( colorsToUpdate => {
+		setAttributes( colorsToUpdate );
+	}, 2000 );
+
+	const setColor = useCallback( colorToUpdate => {
+		setColorsState( state => ( { ...state, ...colorToUpdate } ) );
+		debouncedSetColors( colorToUpdate );
+	}, [] );
+
+	return (
+		<BottomSheet.SubSheet
+			navigationButton={
+				<BottomSheet.Cell
+					label={ __( 'Dynamic Color', 'jetpack-videopress-pkg' ) }
+					onPress={ openSubSheet }
+					leftAlign
+					value={
+						useAverageColor
+							? __( 'On', 'jetpack-videopress-pkg' )
+							: __( 'Off', 'jetpack-videopress-pkg' )
+					}
+				>
+					<Icon icon={ chevronRight }></Icon>
+				</BottomSheet.Cell>
+			}
+			showSheet={ showSubSheet }
+		>
+			<>
+				<BottomSheet.NavBar>
+					<BottomSheet.NavBar.BackButton onPress={ goBack } />
+					<BottomSheet.NavBar.Heading>
+						{ __( 'Dynamic Color', 'jetpack-videopress-pkg' ) }
+					</BottomSheet.NavBar.Heading>
+				</BottomSheet.NavBar>
+				<PanelBody>
+					<ToggleControl
+						label={ __( 'Dynamic color', 'jetpack-videopress-pkg' ) }
+						help={
+							<>
```

Comment:
Curious if adding the `<>` wrapper here is mandatory

Reply:
Oh, thanks for catching that! Removed in https://github.com/Automattic/jetpack/pull/29567/commits/b75ce81cf527dccbdff0b022f9e683c78e5d656a :)